lua/null-ls/builtins/formatting/stylelint.lua

```diff
@@ -16,6 +16,8 @@ return h.make_builtin({
         command = "stylelint",
         args = { "--fix", "--stdin", "--stdin-filename", "$FILENAME" },
         to_stdin = true,
+        from_stderr = true,
+        ignore_stderr = false,
```

Comment:
```suggestion
        ignore_stderr = false,
```

`from_stderr` makes this line superfluous.